### PR TITLE
Fix CommandResult logging & minor clean-ups

### PR DESCRIPTION
**Pull request by vorburger**

### CODE REVIEW

File: api-jvm-impl/src/main/java/ch/vorburger/minecraft/storeys/japi/impl/MinecraftJvmImpl.java [hunk 1]
```diff
@@ -43,15 +41,13 @@
...
-
+    private final PluginInstance plugin;
     private final CommandSource source;
     private final ActionWaitHelper actionWaitHelper;
-    private final PluginInstance plugin;
 
     private final List<Action<?>> actionList = new ArrayList<>();
 
-    MinecraftJvmImpl(CommandSource source, PluginInstance plugin) {
+    MinecraftJvmImpl(PluginInstance plugin, CommandSource source) {
```
Comment (edewit) [inline api-jvm-impl/src/main/java/ch/vorburger/minecraft/storeys/japi/impl/MinecraftJvmImpl.java:50]: Actually atm `CommandSource` isn't really used here

File: api-jvm-impl/src/main/java/ch/vorburger/minecraft/storeys/japi/impl/actions/MainThreadAction.java [hunk 2]
```diff
@@ -38,6 +37,8 @@
  */
 public abstract class MainThreadAction<T> implements Action<T> {
 
+    // private static final Logger LOG = LoggerFactory.getLogger(MainThreadAction.class);
```
Comment (edewit) [inline api-jvm-impl/src/main/java/ch/vorburger/minecraft/storeys/japi/impl/actions/MainThreadAction.java:40]: wouldn't it be better to remove it?

**Review by edewit — CHANGES_REQUESTED** [review]

File: api-jvm-impl/src/main/java/ch/vorburger/minecraft/storeys/japi/impl/MinecraftJvmImpl.java (hunk 1 above)
Comment (vorburger) [reply api-jvm-impl/src/main/java/ch/vorburger/minecraft/storeys/japi/impl/MinecraftJvmImpl.java:50]: How do you mean? We need it to implement `player()`, no?

File: api-jvm-impl/src/main/java/ch/vorburger/minecraft/storeys/japi/impl/actions/MainThreadAction.java (hunk 2 above)
Comment (vorburger) [reply api-jvm-impl/src/main/java/ch/vorburger/minecraft/storeys/japi/impl/actions/MainThreadAction.java]: Yes, done!

**Review by edewit — APPROVED** [review]

**Pull request merged**